### DISCUSSION TITLE
Where to get ucollect.exe for Standalone Client

### DISCUSSION THREAD

**Discussion by deeseem** [fedorovvl/tso_client#258] [Q&A] [answered]
EDIT: Resolved. (Was clicking wrong menu. )Apparently this is needed to enable hightlighting of collectables, maybe similar to the Pinky add-on for browser. I've Googled it, but no download links found so far, and a little vague on what it is or how to actually use the executable if located.

**Comment by adly98** [comment, marked as answer]
why do u need it?
there is a button to collect all collectables on your island in the latest version of the client

**Reply by deeseem** [reply]
Yeah, there is, it's under the "shortcuts/collections" menu. Thanks.
